pkg/fanal/analyzer/sbom/sbom.go

```diff
@@ -29,14 +30,15 @@ var requiredSuffixes = []string{
 
 type sbomAnalyzer struct{}
 
-func (a sbomAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+func (a sbomAnalyzer) Analyze(ctx context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
 	// Format auto-detection
 	format, err := sbom.DetectFormat(input.Content)
 	if err != nil {
 		return nil, xerrors.Errorf("failed to detect SBOM format: %w", err)
 	}
 
-	bom, err := sbom.Decode(input.Content, format)
+	ctx = log.WithContextAttrs(ctx, log.String("file", input.FilePath))
```

Comment:
What if adding `log.FilePath(filePath string)` in `pkg/log` and using it so we can use the consistent key name?

```
func FilePath(filePath string) slog.Attr {
    return String("file_path", filePath)
}
```

Reply:
added (https://github.com/aquasecurity/trivy/pull/7074/commits/5f3f694ca6b2f39e09916a66ba6a22b4293fe453 + https://github.com/aquasecurity/trivy/pull/7074/commits/ae1e4d818c70b65517f5040ffa0f4ad29ba7dd8e)

Reply:
>added (https://github.com/aquasecurity/trivy/commit/5f3f694ca6b2f39e09916a66ba6a22b4293fe453 + https://github.com/aquasecurity/trivy/commit/ae1e4d818c70b65517f5040ffa0f4ad29ba7dd8e)

I think they should be another PR. And merge the change into this PR.

Reply:
You are right. The number of filepaths used was more than i expected.
I created #7080

Reply:
Thanks.  Merged #7080.

Reply:
updated this PR